### DISCUSSION TITLE
JavaScript error

### DISCUSSION THREAD

**Discussion by LucaDev-Ita** [brackets-cont/brackets#281] [Q&A]
**I have a problem**, when I write the JavaScript code in brackets it gives me that all the code is wrong but it is correct because on the devtools console (I use microsoft edge) there are no errors and everything works. [error Image](https://github.com/LucaDev-coder/errorjs/blob/main/bracketsError.png)

**Comment by charlypa** [comment]
could you share the error

**Reply by LucaDev-Ita** [reply]
ok [error Image](https://github.com/LucaDev-coder/errorjs/blob/main/bracketsError.png)

**Reply by charlypa** [reply]
There are missing semicolons. 
JS can work without a semicolon, However, as best practice better add a semicolon.
You can see more details of the error by clicking the exclamation mark in the image 
![bracketsError](https://github.com/brackets-cont/brackets/assets/1525027/ba87fad5-8702-4efc-b615-87e4e9d84fe6)
